nipype/pipeline/plugins/tools.py

```diff
@@ -125,7 +125,14 @@ def create_pyscript(node, updatehash=False, store_exception=True):
     can_import_matplotlib = False
     pass
 
+import os
+value = os.environ.get('NO_NIPYPE_ET', None)
+if value is None:
+    os.environ['NO_NIPYPE_ET'] = "1"
 from nipype import config, logging
+if value is None:
+    del os.environ['NO_NIPYPE_ET']
```

Comment:
I don't really understand this section. When is `create_pyscript` used, and why would you reenable etelemetry?

Reply:
this is used for all HPC execution (SGE, SLURM, etc.,.).